src/facade/dragonfly_listener.cc

```diff
@@ -153,12 +154,12 @@ util::Connection* Listener::NewConnection(ProactorBase* proactor) {
 
 error_code Listener::ConfigureServerSocket(int fd) {
   int val = 1;
-  constexpr int kInterval = 300;  // 300 seconds is ok to start checking for liveness.
 
   if (setsockopt(fd, SOL_SOCKET, SO_REUSEADDR, &val, sizeof(val)) < 0) {
     LOG(WARNING) << "Could not set reuse addr on socket " << SafeErrorMessage(errno);
   }
-  bool success = ConfigureKeepAlive(fd, kInterval);
+  uint32_t interval = absl::GetFlag(FLAGS_tcp_keepalive);
```

Comment:
Nit: I'd remove the var `interval` and call `GetFlag()` inline of `ConfigureKeepAlive()`, your call

Reply:
Actually I need to do validation on this flag value, so register call back is needed.pushing now my changes